python_modules/libraries/dagster-dbt/dagster_dbt_tests/components/code_locations/dependency_on_dbt_project_location/defs/depends_on_dbt_yaml/defs.yaml

```diff
@@ -0,0 +1,7 @@
+type: dependency_on_dbt_project_location.defs.depends_on_dbt_yaml.component.MyExecutableComponent
+
+attributes:
+  assets:
+    - key: downstream_of_customers_two
+      deps:
+        - "{{ context.component_tree.load_component_at_path('jaffle_shop_dbt').get_asset_key_for_model('customers') }}"
```

Comment:
We need to make this less verbose IMO

Either: 

A predefined thing like:

`- "{{ component_at_path('jaffle_shop_dbt').get_asset_key_for_model('customers') }}"`

Or we write a custom udf

`- "{{ jaffle_shop_asset_key('customers') }}"`